data/contributors.json

```diff
@@ -0,0 +1,621 @@
+{
+  "contributors": [
```

Comment:
nit: possible to just make this a flat-level array instead of an object with the `contributors` key?